src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -774,7 +775,7 @@ public HyperwalletBankCard createBankCard(HyperwalletBankCard bankCard) {
             throw new HyperwalletException("Bank Card token may not be present");
         }
         if (bankCard.getType() == null) {
-            bankCard.setType(HyperwalletTransferMethod.Type.BANK_CARD);
+            bankCard.setType(Type.BANK_CARD);
```

Comment:
Change it to HyperwalletBankCard.Type.VENMO_ACCOUNT for better clarity.

Reply:
done

Reply:
Delete this line.

Reply:
Delete this line